qt/draw_widget.hpp

```diff
@@ -86,15 +86,15 @@ public Q_SLOTS:
 private:
   void SubmitFakeLocationPoint(m2::PointD const & pt);
   void SubmitRulerPoint(m2::PointD const & pt);
-  void SubmitRoutingPoint(m2::PointD const & pt);
+  void SubmitRoutingPoint(m2::PointD const & pt, bool pointIsMercator = false);
```

Comment:
No default values please, make it explicit `false /* pointIsMercator */`.

Reply:
Done